app/scripts/modules/core/delivery/executionGroup/execution/execution.component.html

```diff
@@ -0,0 +1,115 @@
+<div class="execution" id="execution-{{::$ctrl.execution.id}}" ng-class="$ctrl.showingDetails ? 'show-details' : 'details-hidden'">
+  <div class="execution-overview group-by-{{::$ctrl.sortFilter.groupBy}}">
+    <h4 class="execution-name" ng-if="::$ctrl.sortFilter.groupBy !== 'name'">
+      <account-label-color ng-repeat="account in $ctrl.execution.deploymentTargets" account="{{account}}"></account-label-color>
+      {{::$ctrl.execution.name}}
+    </h4>
+    <execution-status execution="::$ctrl.execution"
+                      toggle-details="$ctrl.toggleDetails(node)"
+                      showing-details="$ctrl.showingDetails"
+                      standalone="::$ctrl.standalone">
+    </execution-status>
+    <div class="execution-bar">
+      <div class="stages" ng-class="{'show-durations': $ctrl.sortFilter.showStageDuration}">
+        <div ng-repeat="stage in ::$ctrl.execution.stageSummaries"
+             class="clickable stage execution-marker"
+             ng-style="::{width: 100/$ctrl.execution.stageSummaries.length + '%', 'background-color': stage.color}"
+             ng-class="['stage-type-' + stage.type.toLowerCase(), 'execution-marker-' + stage.status.toLowerCase(), $ctrl.isActive($index) ? 'active' : '', stage.isRunning ? 'glowing' : '']"
+             uib-tooltip-template="::stage.labelTemplateUrl"
+             tooltip-title="(Click for details)"
+             tooltip-placement="bottom"
```

Comment:
while you're in here, might as well remove the `tooltip-title` and `tooltip-placement` attributes, since they're not used or doing anything